### DISCUSSION TITLE
I'm getting "^C" on Google Colab.

### DISCUSSION THREAD

**Discussion by davidias244** [TheLastBen/fast-stable-diffusion#730] [Q&A]
Hi. 
When I'm trying to use the Google Colab notebook, the link to access the UI doesn't appear in the end.
I only get this:
LatentDiffusion: Running in eps-prediction mode
DiffusionWrapper has 865.91 M params.
^C

I'm using this notebook on the free version of Google Colab:
https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111-V2.ipynb

Thanks!

**Comment by TheLastBen** [comment]
Disconnect and try again until it works.

**Reply by davidias244** [reply]
I tried 8 times yesterday.
I'll try several more times today. Thanks!

**Reply by davidias244** [reply]
Yeah, I tried several more times and the same thing is happening.

**Reply by davidias244** [reply]
From my little "research" it is probably because there's not enough RAM on the free version of the colab to run SD 2.0.
I think we will just have to wait for a version that uses less RAM, right? 
If that's even possible.

**Reply by ClaraSanders** [reply]
Yes.

**Reply by davidias244** [reply]
I was able to run it twice, but it was like 2 times out of 200.
Do you guys think there will be a fix for that on the future? Or this is it for free colab users?

**Comment by nawnie** [comment]
I get this if I load a faulty ckpt

**Reply by davidias244** [reply]
I don't think that's my problem. I'm loading the default model from the notebook. Thanks!

**Comment by highly-illiquid** [comment]
I'm having the same issue. I've loaded and reloaded multiple times.

**Reply by highly-illiquid** [reply]
<img width="581" alt="Screen Shot 2022-11-29 at 8 39 59 PM" src="https://user-images.githubusercontent.com/7577820/204544610-72ee22d0-524e-42ef-9499-9f7b1b812969.png">

**Reply by ClaraSanders** [reply]
Hi guys, this issue happened to me with joepenna repo and was due to a lack of computing power.

**Reply by ftarnogol** [reply]
does it mean the free colab won't work for v2? I trained v2-512 and even with a "fresh" colab session, i'm still getting the ^C error

**Reply by joanorsky** [reply]
There seems to be some sort of memory leak. Sometimes it passes.. sometimes it crashes due to low RAM...

**Reply by nawnie** [reply]
maybe try the --medram or --lowram argument? havn't messed with 2.0 yet honestly

**Reply by ClaraSanders** [reply]
Since this is the only active topic here can you guys help me with an issue related to the latest Automatic111 update? All the models I merge can only be loaded on this very same repo. When I try to load them on for example relaxed or hlky I get the error: 
KeyError: 'state_dict'
I even downloaded a repo (NMKD Stable Diffusion GUI) on my computer and I get the same thing, only with models merged with the latest update.
If you can't help me, can you tell me at least another colab that merges models like Last Ben's Automatic111? I like to merge the Dreambooth models I make so now I'm screwed. Thanks.
